features/signup.feature

```diff
@@ -0,0 +1,14 @@
+Feature: We are signing up
```

Comment:
I would rephrase this feature title to:
`A visitor can create an account and become a registered user`
or something like that